filament/backend/include/backend/platforms/PlatformEGLAndroid.h

```diff
@@ -102,6 +107,10 @@ class PlatformEGLAndroid : public PlatformEGL {
 
     using clock = std::chrono::high_resolution_clock;
     clock::time_point mStartTimeOfActualWork;
+
+    void* mNativeWindowLib = nullptr;
+    int32_t (*ANativeWindow_getBuffersDefaultDataSpace)(ANativeWindow* window);
```

Comment:
`= nullptr` here to make sure it's initialized?

Reply:
good point, done.